src/ccan/ccan/base64/base64.h

```diff
@@ -2,6 +2,8 @@
 #ifndef CCAN_BASE64_H
 #define CCAN_BASE64_H
 
+#include "config.h"
+#include <stdint.h>
```

Comment:
nit: in the unlikely event we need to update this file then theses changes may get lost. unsure if there's a way of enforcing this in the build somehow.

Reply:
> in the unlikely event we need to update this file then theses changes may get lost.

due to the current structure of this project, it is necessary to include `config.h` (or libsecp256k1-config.h) at the top of the file.
This is done for the other `ccan` header files as well.
As an easy way, can create a tool that checks `#include "config.h"` at building. (Error if not exist)
Auto-generation of source code's can solve the problem, but it takes too much time.
As long as you use only `autotools`, MSVC build is difficult ...